plugins/out_s3/s3.c

```diff
@@ -1126,6 +1156,24 @@ static int s3_put_object(struct flb_s3 *ctx, const char *tag, time_t create_time
     return -1;
 }
 
+int get_md5_base64(char *buf, size_t buf_size, char *md5_str, size_t md5_str_size)
+{
+    unsigned char md5_bin[16];
+
+    int ret = mbedtls_md5_ret(buf, buf_size, md5_bin);
+    if (ret != 0) {
+        return ret;
+    }
+
+    size_t olen;
```

Comment:
I think it should be at the beginning of the function: https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#variable-definitions